### DISCUSSION TITLE
I developed a screenshot program based on Blend2D

### DISCUSSION THREAD

**Discussion by xland** [blend2d/blend2d#172] [Show and tell]
Project Site: https://github.com/xland/ScreenCapture/tree/blend2d-0.1.2

<img width="524" alt="2023888414351" src="https://github.com/blend2d/blend2d/assets/2980915/9dd99a79-b04a-49e2-9041-a3f3cb5832c8">

**Comment by xland** [comment]
Thanks [kobalicek](https://github.com/kobalicek) for his help in developing this program.

**Comment by kobalicek** [comment]
That's really nice, thank you for sharing your work!
